go.mod

```diff
@@ -36,7 +36,7 @@ require (
 	github.com/stretchr/testify v1.8.1
 	github.com/tangzero/inflector v1.0.0
 	github.com/withfig/autocomplete-tools/packages/cobra v1.2.0
-	go.mongodb.org/atlas v0.22.0
+	go.mongodb.org/atlas v0.21.1-0.20230206173512-87e8a19684b0
```

Comment:
NOTE: After v2 gets released version will change to the official release (currently it is points to git commit). 
We cannot use branch name as go tools do not support branches starting with `v.*`